docusaurus.config.js

```diff
@@ -78,29 +79,7 @@ const config = {
           src: "img/icon.png",
           srcDark: "img/iconDark.png",
         },
-        items: [
-          {
-            type: "doc",
-            docId: "index",
-            position: "left",
-            label: "Docs",
-          },
-          {
-            to: "https://mirror.xyz/givepraise.eth",
-            position: "left",
-            label: "Blog",
-          },
-          {
-            href: "/waitlist",
-            position: "left",
-            label: "Get Praise!",
-            className: "navbar-get-praise",
-          },
-          {
-            type: "custom-HeaderSocialLogos",
-            position: "right",
-          },
-        ],
+        items: [],
```

Comment:
Let the header items and social logos stay. New urls:

Docs: /
Blog: https://givepraise.substack.com/
Waitlist: https://givepraise.xyz/waitlist